### ISSUE TITLE
pod install fails

### ISSUE THREAD

**Issue by maka-io** [expo/expo#17340]
### Summary

$ npx pod-install
Scanning for pods...
1.11.3
> pod install
Auto-linking React Native modules for target `ProjectName`: Expo, RNCAsyncStorage, RNCMaskedView, RNCPicker, RNDateTimePicker, RNGestureHandler, RNReanimated, RNScreens, react-native-flipper, and react-native-safe-area-context
Analyzing dependencies
[!] Unable to find a specification for `ExpoModulesCore` depended upon by `Expo`

You have either:
 * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
 * mistyped the name or version.
 * not added the source repo that hosts the Podspec to your Podfile.

### Managed or bare workflow? If you have `ios/` or `android/` directories in your project, the answer is bare!

bare

### What platform(s) does this occur on?

iOS

### SDK Version (managed workflow only)

44

### Environment

  expo-env-info 1.0.3 environment info:
    System:
      OS: macOS 12.3.1
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 14.18.3 - ~/.nvm/versions/node/v14.18.3/bin/node
      npm: 8.5.2 - ~/.nvm/versions/node/v14.18.3/bin/npm
      Watchman: 2022.03.21.00 - /usr/local/bin/watchman
    Managers:
      CocoaPods: 1.11.3 - /usr/local/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 21.4, iOS 15.4, macOS 12.3, tvOS 15.4, watchOS 8.5
    IDEs:
      Android Studio: 2021.1 AI-211.7628.21.2111.8309675
      Xcode: 13.3.1/13E500a - /usr/bin/xcodebuild
    npmPackages:
      expo: ^44.0.6 => 44.0.6 
      react: 17.0.2 => 17.0.2 
      react-dom: 17.0.2 => 17.0.2 
      react-native: 0.67.4 => 0.67.4 
      react-native-web: ^0.17.7 => 0.17.7 
    npmGlobalPackages:
      expo-cli: 5.4.3
    Expo Workflow: bare

### Reproducible demo

`$ npx pod-install`
![Screen Shot 2022-05-04 at 7 11 02 PM](https://user-images.githubusercontent.com/3743882/166839869-1cd4d834-6a5f-4b71-9e78-88985ae7af67.png)

**Comment by byCedric** [comment]
Hi @maka-io! `expo-modules-core` is a direct dependency from `expo`, [you can see that here](https://www.npmjs.com/package/expo?activeTab=dependencies). If it's not installed, but you do have `expo@>=44`, then something must be wrong with your `node_modules` state. 

I would recommend reinstalling it, or validate it with:
- `$ npm list expo-modules-core`
- `$ yarn why expo-modules-core`

Hope this helps!

**Comment by maka-io** [comment]
Thank you @byCedric for the response!

I do not have expo-modules-core installed, and I do have expo@>=44.  I removed my node_modules folder, and reinstalled but the problem persists.

`npm list expo-modules-core` yields:
```
ProjectName@1.0.0 /.../ProjectName
└─┬ expo@44.0.6
  └── expo-modules-core@0.6.5
```

```
$ pod install
Auto-linking React Native modules for target `ProjectName `: Expo, RNCAsyncStorage, RNCMaskedView, RNCPicker, RNDateTimePicker, RNGestureHandler, RNReanimated, RNScreens, react-native-flipper, and react-native-safe-area-context
Analyzing dependencies
[!] CocoaPods could not find compatible versions for pod "Expo":
  In Podfile:
    Expo (from `../node_modules/expo/ios`)

Specs satisfying the `Expo (from `../node_modules/expo/ios`)` dependency were found, but they required a higher minimum deployment target.
```

**Comment by coreybrown89** [comment]
So i'm actually having same issue when trying to build EAS via a github action. Also I am using the managed workflow.

**Comment by coreybrown89** [comment]
@byCedric I can't provide a repo, but `expo-modules-core` is not currently installed (to my knowledge). I was previously using SDK 43 with EAS building in a github action with no issues, after migrating to SDK 44 then to SDK 45 this issue started.

my current github action looks like this

```
name: EAS Build (Beta)
on:
  push:
    branches:
      - develop
  workflow_dispatch:

jobs:
  build:
    name: Install and build
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v2

      - name: Setup Node.js
        uses: actions/setup-node@v2
        with:
          node-version: 16.x
          cache: yarn

      - name: Setup Expo
        uses: expo/expo-github-action@v7
        with:
          expo-version: latest
          eas-version: latest
          token: ${{ secrets.EXPO_TOKEN }}

      - name: Install dependencies
        run: yarn install

      - name: Build on EAS
        run: npx eas-cli build --platform all --profile beta --non-interactive
        env:
          APP_ENV: beta
```

**Comment by MikeCod** [comment]
@byCedric,
I made a demo : https://github.com/MikeCod/Demo
`react-native info` : 
```
System:
    OS: macOS 11.6.2
    CPU: (4) x64 Intel(R) Core(TM) i7-5557U CPU @ 3.10GHz
    Memory: 493.68 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.15.1 - /usr/local/bin/node
    Yarn: Not Found
    npm: 8.11.0 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.11.3 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.2, iOS 15.2, macOS 12.1, tvOS 15.2, watchOS 8.3
    Android SDK: Not Found
  IDEs:
    Android Studio: 2021.2 AI-212.5712.43.2112.8609683
    Xcode: 13.2.1/13C100 - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.15 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: ^18.1.0 => 18.1.0 
    react-native: ^0.68.2 => 0.68.2 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```

Let me know your advancement please !

**Comment by prasanna-col** [comment]
cmd: eas build --profile development --platform ios

**install pod failed in expo when I try to generate the development build.!**


<img width="520" alt="image" src="https://user-images.githubusercontent.com/98028568/178760600-9ce13b0d-a213-4736-a114-821aa1d0c6aa.png">

Does Anyone have a solution?

**Comment by samih-sghier** [comment]
any update on this?

**Comment by kubmir** [comment]
Having the same issue for `expo-notifications` Somebody with any hack here ?

**Comment by Yassdrk** [comment]
same issue....... 
![sa](https://user-images.githubusercontent.com/82156431/215907225-0c426cbf-a0c3-491e-8842-71b43db1e78e.png)

**Comment by genomanetwork** [comment]
Same Issue ....

![Captura de pantalla 2023-10-06 a la(s) 11 13 47](https://github.com/expo/expo/assets/28502983/cf52ff92-88fc-4c07-9162-f9df8624dd8e)

**Comment by kongwykon** [comment]
still crashing here

**Comment by anatoleblanc** [comment]
Same issue for me :'(

**Comment by rickithadi** [comment]
:(

**Comment by roni-castro-shipt** [comment]
The fix for me with "expo": "^52.0.37" and "react-native": "0.76.7" was to set the minimum ios version to 15.1 mentioned [here](https://reactnative.dev/blog/2024/10/23/release-0.76-new-architecture#updates-to-minimum-ios-and-android-sdk-requirements) 
```js
platform :ios, 15.1
```
After that, I deleted my `ios/Pods` cache and ran pod install and it stopped giving the error `Unable to find a specification for ExpoModulesCore depended upon by Expo`
